windows/guest_os_inplace_upgrade/guest_os_inplace_upgrade.yml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# This test case is testing guest OS in-place upgrade.
+# Covered scenarios:
+# 1. When there is no inbox pvscsi and vmxnet3 drivers in the OS upgraded from,
```

Comment:
there is ---> there are
"in the OS upgraded _**from**_"?

Reply:
Thanks, this 'Covered scenarios' part is not needed.